src/nvim/version.c

```diff
@@ -939,21 +939,11 @@ void list_version(void)
   version_msg(USR_VIMRC_FILE2);
   version_msg("\"\n");
 #endif  // ifdef USR_VIMRC_FILE2
-#ifdef USR_VIMRC_FILE3
-  version_msg(_(" 3rd user vimrc file: \""));
-  version_msg(USR_VIMRC_FILE3);
-  version_msg("\"\n");
-#endif  // ifdef USR_VIMRC_FILE3
 #ifdef USR_EXRC_FILE
   version_msg(_("      user exrc file: \""));
   version_msg(USR_EXRC_FILE);
   version_msg("\"\n");
 #endif  // ifdef USR_EXRC_FILE
-#ifdef USR_EXRC_FILE2
-  version_msg(_("  2nd user exrc file: \""));
-  version_msg(USR_EXRC_FILE2);
-  version_msg("\"\n");
-#endif  // ifdef USR_EXRC_FILE2
```

Comment:
If this is going to be removed then the docs should be updated as well. Anyways, I'm not actually sure it should be removed... I remember @justinmk mentioned it but I don't think it's been discussed much.